src/backend/distributed/citus--9.4-1--9.5-1.sql

```diff
@@ -0,0 +1,4 @@
+/* citus--9.4-1--9.5-1 */
```

Comment:
We started to disallow c-style comments. 
```suggestion
-- citus--9.4-1--9.5-1
```